src/ConnectedMode/Migration/Wizard/MigrationWizardController.cs

```diff
@@ -84,31 +82,15 @@ public void StartMigrationWizard(BoundSonarQubeProject oldBinding)
             }
         }
 
-        private Action GetTfVcHelpAction()
```

Comment:
I didn't want to use tfsHelpAction existence to determine if I should show the checkbox.

Reply:
You also removed logging which could be useful. Was that intentional?

Reply:
Yes we already log if git is not found. I do not believe these logs add much value so I didn't add those in the new method.